app/Http/Controllers/Backend/MediaWikiHostController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers\Backend;
+
+use App\Http\Controllers\Controller;
+use App\Services\MediaWikiHostResolver;
+use App\Services\UnknownDBVersionException;
+use App\Services\UnknownWikiDomainException;
+use Illuminate\Http\Request;
+
+class MediaWikiHostController extends Controller {
+    public function getWikiHostForDomain(Request $request): \Illuminate\Http\JsonResponse {
```

Comment:
nit-pick (i.e. non-blocking): didn't we already use this? Could therefore not but full path to class